### ISSUE TITLE
Functional dashboard page for website

### ISSUE THREAD

**Issue by catherineunciano** [IT3A-2016/Paper-Icebergs#32]

**Comment by catherineunciano** [comment]
<!DOCTYPE html>
<html>

<head>
	<title>QWS</title>
	
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1, description" content="">
	<meta name="author" content="">
	
	<link rel="stylesheet" href="../css/bootstrap.min.css" type="text/css">
	<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
	
	<script src="../js/smoothscroll.js"></script>
	<script src="../js/webscript.js"></script>
	<script src="../js/jquery.js"></script>
	<script src="../js/bootstrap.min.js"></script>
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
	<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
	
	<style>
	body {background-color:#2E86C1;}
	h1   {color: blue;}
	p    {color: red;}
	div.container {
		margin-left: 170px;
		border-left: 1px solid gray;
		padding: 1em;
		overflow: hidden;
	}
	</style>
</head>
<body class="cusbodycolor" oncontextmenu="return false">
	<div class="row">
		<div class="col-md-12 well">
			<div class="col-sm-8">
				<span style="font-family:'Times New Roman', Times, serif;font-size:20px;"/>Automated Queueing Web-based System
			</div>
			<div class="col-sm-4"  style="text-align:center;">
				<a href="logout.php"><span style="font-family:'Times New Roman', Times, serif;font-size:16px;">Sign Out</span></a><br><br>		
			</div>
		</div>
	</div>
		
	<div id="page-wrapper">
		<div class="row">
			<div class="col-sm-12"  style="padding:20px 0 20px 0;text-align:center;"><br><br></div>
		</div>
		<div class="container-fluid">
		
			<div class="col-lg-10 col-lg-offset-1 well">
				
				<div class="row">
					<div class="col-sm-12"  style="padding:20px 0 20px 0;text-align:center;"><br><br></div>
				</div>
				
				<div class="col-md-6">
					<div class="col-sm-6">
						<div style="text-align:center;padding:10px 0 10px 0;background-color:#AED6F1;">
							<span class><br><br>Current Serving<br><br><br></span>
						</div><br>
					</div>	
					
					<div class="col-sm-6"> 
						<div style="text-align:center;padding:10px 0 10px 0;background-color:#AED6F1;">
							<span class><br><br>Code<br><br><br></span>
						</div><br>	
					</div>
						
					<div class="row">
						<div class="col-sm-12"><br>
							<div class="col-sm-12">
								<div style="text-align:center;padding:10px 0 10px 0;background-color:#FF7000;">
									End
								</div><br>
							</div>
							<div class="col-sm-12">
								<div style="text-align:center;padding:10px 0 10px 0;background-color:#00FF00;">
									Next
								</div><br>
							</div>
							<div class="col-sm-12">
								<div style="text-align:center;padding:10px 0 10px 0;background-color:#FFFF00;">
									Repeat Call
								</div><br>
							</div>
							<div class="col-sm-12">
								<div style="text-align:center;padding:10px 0 10px 0;background-color:#FF0000;">	
									No Show
								</div><br>
							</div>
							<div class="col-sm-12">
								<div style="text-align:center;padding:10px 0 10px 0;background-color:#FF8F8F;">	
									Stop Queue
								</div><br>
							</div>
							<div class="col-sm-12">
								<div style="text-align:center;padding:10px 0 10px 0;background-color:#AADBE7;">
									Reset Queue
								</div><br>
							</div>
						</div>
					</div>
				</div>
				
				<div class="col-md-6">
					<div class="row">
						<div class="col-sm-6">
							<div style="text-align:center;padding:10px 0 500px 0;background-color:#AED6F1;">Reserved Numbers</div>
						</div>
						<div class="col-sm-6">
							<div div style="text-align:center;padding:10px 0 500px 0;background-color:#AED6F1;">Its Unique Code</div>
						</div>
					</div>
				</div>
				
				<div class="row">
					<div class="col-sm-12"  style="padding:20px 0 20px 0;text-align:center;"><br><br></div>
				</div>
				
			</div>
		</div>
	</div>
	
	<div class="row" style="padding:20px 0 20px 0;"><br><br></div>
	<footer style="text-align:center;background-color:#707B7C;color:#000000;">Paper-Icebergs 2017 &copy; All rights reserved.</footer>
	
</body>
</html>

**Comment by catherineunciano** [comment]
![dash.jpg](https://zube.io/files/IT3A-2016/8a9355ad737c63e3d5e34f5d7a470efd.jpeg)

I zoom-out it, so I can show the whole page. :)
